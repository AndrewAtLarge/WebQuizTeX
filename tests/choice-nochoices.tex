% should give error as has no \incorrect or \correct inside choice
\documentclass{webquiz}
\title{A multiple choice question with a unique answer}
\begin{document}
  \begin{question}     % a quiz question
     What is the cube root of $64$?
     \begin{choice}[columns=3]% unique answer rendered in three columns
     \end{choice}
  \end{question}
\end{document}
