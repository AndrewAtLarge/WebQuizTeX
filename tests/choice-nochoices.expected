! Package webquiz Error: Choice environment with no choices!.
